mock-api/msw/handlers.ts

```diff
@@ -1601,26 +1610,76 @@ export const handlers = makeHandlers({
     return handleOxqlMetrics(params.body)
   },
   siloMetric: handleMetrics,
+  affinityGroupList: ({ query }) => {
+    const project = lookup.project({ ...query })
+    const affinityGroups = db.affinityGroups.filter((i) => i.project_id === project.id)
+    return paginated(query, affinityGroups)
+  },
+  affinityGroupView: ({ path, query }) => lookup.affinityGroup({ ...path, ...query }),
+  affinityGroupMemberList: ({ path, query }) => {
+    const project = lookup.project({ ...query })
+    const affinityGroup = lookup.affinityGroup({
+      project: project.id,
+      affinityGroup: path.affinityGroup,
+    })
```

Comment:
this one can also be

```suggestion
    const affinityGroup = lookup.affinityGroup({ ...path, ...query })
```